langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/statements/foreach/ForeachErrorHandlingTests.java

```diff
@@ -53,7 +53,8 @@ public void testArrayForeachAndTrap() {
             "error: \\{ballerina\\}TypeCastError \\{\"message\":\"incompatible types: 'error' cannot be cast to " +
                     "'int'\"\\}\n" +
                     "\tat foreach_error_handling:\\$lambda\\$_0\\(foreach_error_handling.bal:41\\)\n" +
-                    "\t   foreach_error_handling:\\$lambda\\$_0\\$lambda0\\$\\(foreach_error_handling.bal:40\\)")
+                    "\t   foreach_error_handling:\\$lambda\\$_0\\$lambda0\\$\\(foreach_error_handling.bal:40\\)",
+            enabled = false)
```

Comment:
Please create an issue to track this.

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/27361